hadoop-hdds/rocks-native/src/main/java/org/apache/hadoop/hdds/utils/db/managed/ManagedSSTDumpIterator.java

```diff
@@ -126,11 +135,18 @@ private Optional<UnsignedLong> getNextUnsignedLong() throws IOException {
   }
 
   private void init(ManagedSSTDumpTool sstDumpTool, File sstFile,
-                    ManagedOptions options)
+      ManagedOptions options, Optional<ManagedSlice> lowerKeyBound,
+      Optional<ManagedSlice> upperKeyBound)
       throws NativeLibraryNotLoadedException {
-    String[] args = {"--file=" + sstFile.getAbsolutePath(), "--command=scan",
-        "--silent"};
-    this.sstDumpToolTask = sstDumpTool.run(args, options);
+    Map<String, String> argMap = Maps.newHashMap();
+    argMap.put("file", sstFile.getAbsolutePath());
+    argMap.put("silent", null);
```

Comment:
QQ:
1. What's the default value for this parameter? What happens if you don't add it to `argMap`?
1. What does it do?

Reply:
Without --silent adds some unwanted logs in SSTDumpTool. It should have a value. You can look at ManagedSSTDumpTool class.